fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -238,8 +242,12 @@ public DataApiRequest(
 
         this.having = DruidHavingBuilder.buildHavings(this.havings);
 
+        this.dateTimeSort = generateDateTimeSortColumn(sorts);
+
+        String sortApiValue = generateApiSortValue(sorts);
+
```

Comment:
I wouldn't call this 'generate...'  Maybe 'filterTimeSort' or 'trimTimeSort'.

Then I'd consider putting this inline in the call to generateSortColumns(...) and save a local variable